custom-fonts.md

```diff
@@ -139,3 +139,267 @@ object specifies a weight or style for which is there is no exact font file, the
 engine uses one of the more generic files for the font and attempts to
 extrapolate outlines for the requested weight and style.
 
+
+## Example
+Here is an example delcaring and using the fonts in a simple application.
+
+| iOS | Android |
+|------|------|
+| <img style="width: 200px;" src="images/fonts-example-ios.png" /> | <img style="width: 200px;" src="images/fonts-example-android.png" /> |
+
+
+Declaring the fonts in the pubsec.yaml.
+
+```yaml
+name: my_application
+description: A new Flutter project.
+
+dependencies:
+  flutter:
+    sdk: flutter
+    
+flutter:
+  # Include the Material Design fonts.
+  uses-material-design: true
+
+  fonts:
+    - family: Rock Salt
+      fonts:
+        # https://fonts.google.com/specimen/Rock+Salt
+        - asset: fonts/RockSalt.ttf
+    - family: VT323
+      fonts:
+        # https://fonts.google.com/specimen/VT323
+        - asset: fonts/VT323-Regular.ttf
+    - family: Ewert
+      fonts:
+        # https://fonts.google.com/specimen/Ewert
+        - asset: fonts/Ewert-Regular.ttf
+```
+
+Implementing the fonts in the application source code. 
+
+```dart
+import 'package:flutter/material.dart';
+
+const String words1 = "Almost before we knew it, we had left the ground.";
```

Comment:
next time, you don't have to specify "String". `const words1` would be sufficient.